lib/api/_loaders/_command-loader.js

```diff
@@ -33,8 +34,9 @@ class BaseCommandLoader extends BaseLoader {
   getCommandOptions() {
     const redact = this.module.RedactParams || false;
     const alwaysAsync = this.module.alwaysAsync || false;
+    const isTraceable = (Utils.isFunction(this.module.isTraceable)? this.module.isTraceable() : this.module.isTraceable) || false ;
```

Comment:
```suggestion
    const isTraceable = this.module.isTraceable;
```

Reply:
isTraceable should be added on the base command and protocol command classes.

Reply:
`Utils.isFunction(this.module.isTraceable)` is for custom commands which are not classes. Do we remove that also?

Reply:
No, but those can also use a property.

Reply:
```
module.exports.command = function() {...}
module.exports.isTraceable = true;
```